package.json

```diff
@@ -92,7 +92,7 @@
     "@mui/internal-markdown": "^1.0.10",
     "@mui/internal-test-utils": "^1.0.8",
     "@mui/material": "^5.16.7",
-    "@mui/monorepo": "github:mui/material-ui#55bea65c83c1beac77382fe961f8aa72eec21daa",
+    "@mui/monorepo": "github:alexfauquette/material-ui#926c6ccf0ad73929e92b1dd14b9081238eb9a7fe",
```

Comment:
So smooth. 

@Janpot I guess with the process where we release npm package, we will use Codesandbox CI for the same experience?

Reply:
> we will use Codesandbox CI for the same experience?

Yes, that's the idea